bin/generate-version.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+CURL_LATEST_RELEASE_VERSION="curl -s https://api.github.com/repos/Kinto/kinto-admin/releases/latest | jq -r .tag_name"
+KINTO_ADMIN_VERSION_FALLBACK=$(git describe --tags --abbrev=4) || eval "$CURL_LATEST_RELEASE_VERSION"
+
+if [ -n "$KINTO_ADMIN_VERSION" ]; then
+    VERSION="${KINTO_ADMIN_VERSION}"
+else
```

Comment:
When does this happen? Won't there always be a _latest_ version on the releases page?

Reply:
> Won't there always be a latest version on the releases page?

Yes, but we want to account for the case where someone wants to build a version that isn't the latest version.

This would be useful in a script like one of our `build-kinto-admin` scripts. Since there's no `git` directory when we `curl -OL "https://github.com/Kinto/kinto-admin/archive/refs/tags/${TAG}.tar.gz"`, having this env var will allow us to inject the correct version (since it won't be able to read from the tag).